lib/AstOfLlbc.ml

```diff
@@ -172,6 +213,25 @@ module RustNames = struct
   let from =
     parse_pattern "core::convert::From<@T, @U>::from"
 
+  let into_u16 =
```

Comment:
@W95Psp I heard from @sonmarcho that there is a better, more concise way to do this, so this would be another interesting mini-task to add onto this PR -- refactor this bit to have more effective matching logic here, and in the main expression translation function

Reply:
What I do is actually quite simple, see [this](https://github.com/AeneasVerif/aeneas/blob/dc8ceedffa0945b84f81e318fa78f1e55b1731d2/compiler/ExtractBuiltin.ml#L13) and [this](https://github.com/AeneasVerif/aeneas/blob/dc8ceedffa0945b84f81e318fa78f1e55b1731d2/compiler/ExtractBuiltin.ml#L288)

Reply:
Is there a way to recover the values for which @T and @U matched? I guess that's what I had in mind when I wrote "a better way"